server/api/core/core.js

```diff
@@ -163,6 +163,12 @@ export default {
     return shop && shop._id;
   },
 
+  getShopOwner(vendorId) {
+    const shop = Shops.findOne({vendorId: vendorId});
+    if (shop === undefined) return Shops.findOne();
```

Comment:
You can do this `!shop` instead. As it covers for empty string, empty object, other falsy values and it is a brief.